library-valcon-internship/src/components/Header/Header.css

```diff
@@ -31,19 +32,39 @@
 .header-user {
   width: 10%;
   display: flex;
-  justify-content: space-around;
+  justify-content: center;
   align-items: center;
 }
 
 .header-btn {
+  display: flex;
+  height: 1.563rem;
+  width: 4.688rem;
```

Comment:
Is there any reason why this is so particular?

Reply:
used a site for px to rem converting and it gave those numbers, I can change them to a more general value.

Reply:
Then why choose rem at all?